### ISSUE TITLE
Confirmation still shown from a record view on an ModelView action

### ISSUE THREAD

**Issue by Th4nat0s** [dpgaspar/Flask-AppBuilder#1529]
Hello, I got the following issue on my env 👍 

### Environment
Flask==1.1.1
Flask-AppBuilder==3.1.1
Flask-Babel==1.0.0
Flask-JWT-Extended==3.25.0
Flask-Login==0.4.1
Flask-Mail==0.9.1
Flask-OpenID==1.2.5
flask-skeleton==1.0.1
Flask-SQLAlchemy==2.4.1
Flask-User==1.0.2.2
Flask-WTF==0.14.3
marshmallow-sqlalchemy==0.23.1
mysqlclient==1.3.10
SQLAlchemy==1.3.15
SQLAlchemy-Utils==0.36.8
sqlparse==0.2.4

### Describe the expected results
I use a custom action on a Modelview.

```python
class DbView(ModelView):
    @action("myaction", "Json", confirmation=None , multiple=True)
    def myaction(self, item):
        [some code]
```

When confirmation is set to NONE I have no confirmation if I use the action from the list view. Everything is working fine.
![image](https://user-images.githubusercontent.com/1728186/100880711-5d0c3600-34ad-11eb-9c79-4da5303a2c7a.png)

BUT.. On the Item view, If I use the test button I still have a confirmation request with an empty message.
![image](https://user-images.githubusercontent.com/1728186/100880944-a78db280-34ad-11eb-9794-26577ecca104.png)

![image](https://user-images.githubusercontent.com/1728186/100880295-d6efef80-34ac-11eb-9ffd-c018d4c741af.png)

**Comment by dpgaspar** [comment]
Hi,

Thank you for reporting it. A fix is on it's way

**Comment by dpgaspar** [comment]
merged, should be included on 3.1.2
